payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -221,205 +206,207 @@ class LpmRepository constructor(
-                sharedDataSpec.selectorIcon?.darkThemePng,
-                false,
-                PaypalRequirement,
-                LayoutSpec(sharedDataSpec.fields)
-            )
-            PaymentMethod.Type.Affirm.code -> SupportedPaymentMethod(
-                "affirm",
-                false,
-                R.string.stripe_paymentsheet_payment_method_affirm,
-                R.drawable.stripe_ic_paymentsheet_pm_affirm,
-                sharedDataSpec.selectorIcon?.lightThemePng,
-                sharedDataSpec.selectorIcon?.darkThemePng,
-                false,
-                AffirmRequirement,
-                LayoutSpec(sharedDataSpec.fields)
-            )
-            PaymentMethod.Type.RevolutPay.code -> SupportedPaymentMethod(
-                "revolut_pay",
-                false,
-                R.string.stripe_paymentsheet_payment_method_revolut_pay,
-                R.drawable.stripe_ic_paymentsheet_pm_revolut_pay,
-                sharedDataSpec.selectorIcon?.lightThemePng,
-                sharedDataSpec.selectorIcon?.darkThemePng,
-                false,
-                RevolutPayRequirement,
-                LayoutSpec(sharedDataSpec.fields)
-            )
-            PaymentMethod.Type.MobilePay.code -> SupportedPaymentMethod(
-                "mobilepay",
-                false,
-                R.string.stripe_paymentsheet_payment_method_mobile_pay,
-                R.drawable.stripe_ic_paymentsheet_pm_mobile_pay,
-                sharedDataSpec.selectorIcon?.lightThemePng,
-                sharedDataSpec.selectorIcon?.darkThemePng,
-                false,
-                MobilePayRequirement,
-                LayoutSpec(sharedDataSpec.fields)
-            )
-            PaymentMethod.Type.AuBecsDebit.code -> SupportedPaymentMethod(
-                "au_becs_debit",
-                true,
-                R.string.stripe_paymentsheet_payment_method_au_becs_debit,
-                R.drawable.stripe_ic_paymentsheet_pm_bank,
-                sharedDataSpec.selectorIcon?.lightThemePng,
-                sharedDataSpec.selectorIcon?.darkThemePng,
-                true,
-                AuBecsDebitRequirement,
-                LayoutSpec(sharedDataSpec.fields)
-            )
-            PaymentMethod.Type.USBankAccount.code -> SupportedPaymentMethod(
-                "us_bank_account",
-                true,
-                R.string.stripe_paymentsheet_payment_method_us_bank_account,
-                R.drawable.stripe_ic_paymentsheet_pm_bank,
-                sharedDataSpec.selectorIcon?.lightThemePng,
-                sharedDataSpec.selectorIcon?.darkThemePng,
-                true,
-                USBankAccountRequirement,
+    private fun convertToSupportedPaymentMethod(
+        @Suppress("UNUSED_PARAMETER") stripeIntent: StripeIntent,
+        sharedDataSpec: SharedDataSpec,
+    ) = when (sharedDataSpec.type) {
+        PaymentMethod.Type.Card.code -> SupportedPaymentMethod(
+            "card",
```

Comment:
Nit: If we’re already having this diff, can we use the opportunity to add named arguments?

Reply:
It's a whitespace only change. I figured it would be easier to review that way. I can do it here, or in another quick follow up.